packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -3069,6 +3071,7 @@ function beginWork(
     if (
       oldProps !== newProps ||
       hasLegacyContextChanged() ||
+      (current.effectTag & ForceUpdateForLegacySuspense) !== NoEffect ||
```

Comment:
Kinda sucks to have this in the hot path. Not sure what else to do.

Reply:
If you mark an update on `fiber.lanes`, that will force it into the `else`  block.

Reply:
Don't we already do it? https://github.com/facebook/react/blob/f4097c1aef173ea0cb873bc18b47d1ef12bab4b3/packages/react-reconciler/src/ReactFiberThrow.old.js#L277

Doesn't seem to help.

Reply:
Ohh I see.